scout/server/blueprints/variant/controllers.py

```diff
@@ -442,81 +488,50 @@ def observations(store, loqusdb, case_obj, variant_obj):
         variant_obj (scout.models.Variant)
 
     Returns:
-        obs_data(dict)
+        obs_data(dict) with loqusdb id as key and observations stats as value
     """
+    obs_data = {}
     institute_id = variant_obj["institute"]
     institute_obj = store.institute(institute_id)
-    loqusdb_id = institute_obj.get("loqusdb_id") or "default"
-    if loqusdb.loqusdb_settings[loqusdb_id]["version"] is None:
-        flash("Could not connect to the preselected loqusdb instance", "danger")
-        return {
-            "total": "N/A",
-        }
 
-    chrom = variant_obj["chromosome"]
```

Comment:
I've removed all this code used to create the loqusdb query from here and I've moved it to the loqusdb extension instead.